besu/src/test/java/org/hyperledger/besu/cli/PublicKeySubCommandTest.java

```diff
@@ -113,9 +113,9 @@ public void callingPublicKeyExportSubCommandHelpMustDisplayUsage() {
 
   @Test
   public void callingPublicKeyExportSubCommandWithoutPathMustWriteKeyToStandardOutput() {
-    final KeyPair keyPair = KeyPair.generate();
+    final KeyPair keyPair = getKeyPair();
 
-    parseCommand(f -> keyPair, PUBLIC_KEY_SUBCOMMAND_NAME, PUBLIC_KEY_EXPORT_SUBCOMMAND_NAME);
+    parseCommand(PUBLIC_KEY_SUBCOMMAND_NAME, PUBLIC_KEY_EXPORT_SUBCOMMAND_NAME);
 
     final String expectedOutputStart = keyPair.getPublicKey().toString();
```

Comment:
can't you do this check on the NodeKey, rather than the KeyPair? I.e. these tests shouldn't need to know about keypair

Reply:
yes, now exposing NodeKey from AbstractCommandTest instead of KeyPair.